README.md

```diff
@@ -58,6 +66,41 @@ spec:
             - Resource
 ```
 
+### Artifact Hub
+
+Add an `artifacthub-pkg.yml` metadata file to the folder. See an example metadata file for Kyverno policies below and customize per the comments.
+
+```yaml
+---
+name: backup-all-volumes # The name of the package (only alphanum, no spaces, dashes allowed)
+version: 1.0.0 # Version of the policy
+displayName: Backup All Volumes  # Display name of the policy
+createdAt: "2023-03-29T00:00:00.000Z" # The date this package was created (RFC3339 layout)
+description: >-
```

Comment:
Please add the actual description here just like you've done below for the `readme` field.